apps/labs/posts/portable-python-env.md

```diff
@@ -0,0 +1,177 @@
+---
+title: 'Creating a Portable Python Environment from Imports'
+published: June 30, 2020
+author: kim-pevey
+description: |
```

Comment:
@noatamir I wanted to highlight this change specifically, so you're aware that a string in this yaml section cannot run across multiple lines in the file unless you use the `|`.

Reply:
Thanks!